### PR TITLE
add VM lifecycle tests

### PR DESCRIPTION
**Pull request by dshchedr**
**What this PR does / why we need it**:  Adds VM lifecycle tests

Tests covers:

Create/Delete VM via oc and kubectl
Control VM via virtctl (start, connect to console, VNC, check IP)
Negative test: delete non existing VM, create VM with existing name

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/manifests/vm-fedora-template.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
```
Comment (ILpinto) [inline tests/manifests/vm-fedora-template.yml]: You can modify /tests/manifests/virt-testing-vm.yml
And add the cloud init part (it parameter anyway) 
no need for new file.

Comment (dshchedr) [reply tests/manifests/vm-fedora-template.yml]: /tests/manifests/virt-testing-vm.yml - is template for VMI, but in my tests I need to create VM (for verifying start/stop.. etc)

may be it's better to rename virt-testing-vm.yml to virt-testing-vmi.yml because its name is a little confusing.. and rename my vm-fedora-template.yml to virt-testing-vm.yml for future needs

Comment (dshchedr) [reply tests/manifests/vm-fedora-template.yml]: I updated virt-testing-vm.yml, now it creates VM instead of VMI. 
Also, had to update high_performance_vm_test.go

**Comment by ILpinto** [comment]
LGTM

File: tests/framework/types.go [hunk 2]
```diff
@@ -39,22 +39,22 @@ type VirtualMachine struct {
 
 func (vm VirtualMachine) Create() (string, string, error) {
 	args := []string{"create", "-f", vm.Manifest}
-	return ktests.RunCommandWithNS(vm.Namespace, ktests.KubeVirtOcPath, args...)
+	return ktests.RunCommandWithNS(vm.Namespace, "oc", args...)
```
Comment (myakove) [inline tests/framework/types.go]: Why did you change ktests.KubeVirtOcPath to "oc", if from some reason the command will change it's easier to change it in one place 'ktests.KubeVirtOcPath'.

File: tests/framework/util.go [hunk 3]
```diff
@@ -68,6 +68,17 @@ func RunOcDescribeCommand(resourceType, resourceName string) string {
 	return execute(Result{cmd: "oc", verb: "describe", resourceType: resourceType, resourceName: resourceName})
 }
 
+func GetVirtualMachineSpecificParameters(resourceType, resourceName, query string) string {
```
Comment (myakove) [inline tests/framework/util.go]: Is this only for VM?

Comment (dshchedr) [reply tests/framework/util.go]: renamed to GetResourceSpecificParameters as It is necessary not only for VM

File: tests/framework/types.go (hunk 2 above)
Comment (dshchedr) [reply tests/framework/types.go]: created **oc** and **virtctl** as constants

**Comment by dshchedr** [comment]
added to tier1
